src/Glpi/Application/View/TemplateRenderer.php

```diff
@@ -196,11 +238,36 @@ public function display(string $template, array $variables = []): void
      */
     public function renderFromStringTemplate(string $template, array $variables = []): string
     {
+        if ($this->inline_environment === null) {
+            $this->inline_environment = $this->getNewEnvironment();
+        }
+
         try {
             Profiler::getInstance()->start($template, Profiler::CATEGORY_TWIG);
-            return $this->environment->createTemplate($template)->render($variables);
+            return $this->inline_environment->createTemplate($template)->render($variables);
```

Comment:
Maybe rely on a `getEnvironment($inline = true)` or a `getInlineEnvironment()` to "match" other changes.